components/org.wso2.carbon.uis/src/main/java/org/wso2/carbon/uis/internal/CarbonUiServer.java

```diff
@@ -57,10 +52,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CarbonUiServer.class);
 
-    private final AppRegistry appRegistry = new AppRegistry();
-    private final RequestDispatcher requestDispatcher = new RequestDispatcher();
     private HttpConnector httpConnector;
-    private BundleContext bundleContext;
+    private final ConcurrentMap<String, App> deployedApps = new ConcurrentHashMap<>();
```

Comment:
What's the key, what's the value? Please add a comment explaining the map.

Reply:
fixed in commit 604edf199af8547c31b24bd412a169163c7c028c